pkg/daemon/server/service/pipeline_metrics_query.go

```diff
@@ -91,13 +91,13 @@ func (ps *pipelineMetadataQuery) ListBuffers(ctx context.Context, req *daemon.Li
 	resp := new(daemon.ListBuffersResponse)
 
 	buffers := []*daemon.BufferInfo{}
-	for _, buffer := range ps.pipeline.GetAllBuffers() {
-		bufferInfo, err := ps.isbSvcClient.GetBufferInfo(ctx, buffer)
+	for _, buffer := range ps.pipeline.GetAllPartitions() {
```

Comment:
I think we should change "buffers" and "buffer" as well

Reply:
unless you prefer to do this with multiple PRs - I feel like when this effort is done there should probably only be a few uses of the word "buffer" left in the code altogether - do you agree or no?